lib/ansible/modules/network/vyos/vyos_facts.py

```diff
@@ -1,65 +1,91 @@
 #!/usr/bin/python
-#
-# This file is part of Ansible
-#
-# Ansible is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# Ansible is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
-#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The module file for vyos_facts
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
 ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
+                    'status': [u'preview'],
                     'supported_by': 'network'}
 
 
 DOCUMENTATION = """
 ---
 module: vyos_facts
-version_added: "2.2"
-author: "Nathaniel Case (@Qalthos)"
-short_description: Collect facts from remote devices running VyOS
+version_added: 2.2
+short_description: Get facts about vyos devices.
 description:
-  - Collects a base set of device facts from a remote device that
-    is running VyOS.  This module prepends all of the
-    base network fact keys with U(ansible_net_<fact>).  The facts
-    module will always collect a base set of facts from the device
-    and can enable or disable collection of additional facts.
+  - Collects facts from network devices running the vyos operating
+    system. This module places the facts gathered in the fact tree keyed by the
+    respective resource name.  The facts module will always collect a
+    base set of facts from the device and can enable or disable
+    collection of additional facts.
+author: Nilashish Chakraborty (@nilashishc)
 extends_documentation_fragment: vyos
 notes:
-  - Tested against VYOS 1.1.7
+   - Tested against VyOS 1.2.0
```

Comment:
We should also indicate 1.1.8 here, or wait on merging until we have 1.2.x online.